File: app/src/main/java/vn/tiki/sample/util/Urls.java
package vn.tiki.sample.util;

import android.support.annotation.NonNull;
import vn.tiki.sample.BuildConfig;

public final class Urls {

  @NonNull
  public static String resolveImageUrl(String imageUrl) {
    return BuildConfig.BASE_URL + imageUrl;
  }


  private Urls() {
    throw new InstantiationError();
  }

}

File: sample/src/main/java/vn/tiki/sample/util/Urls.kt
package vn.tiki.sample.util

import vn.tiki.sample.BuildConfig

class Urls private constructor() {


  init {
    throw InstantiationError()
  }

  companion object {
    @JvmStatic
    fun resolveImageUrl(imageUrl: String): String {
      return BuildConfig.BASE_URL + imageUrl
    }
  }

}
